components/camera/client.go

```diff
@@ -217,8 +168,8 @@ func (c *client) Stream(
 				}
 				return
 			case frameCh <- gostream.MediaReleasePairWithError[image.Image]{
-				Media:   frame,
-				Release: release,
+				Media:   img,
+				Release: func() {},
```

Comment:
So release is now a no-op if using client stream?

Reply:
I thiiink it always has been a no-op on these single frame retrievals

If you take a look at the now-deleted `Read` it returns an empty anon func at the end